handler.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: all
+  gather_facts: false
+  tasks:
+    - name: non-idempotent task
+      shell: |
```

Comment:
That's an unusual syntax. I'd suggest `command: /bin/true`. And then force the handler by doing `changed_when: true`